### ISSUE TITLE
"Failed to check system storage" (+potential performance issues?)

### ISSUE THREAD

**Issue by Fivavoa** [TopazTK/KH-ReFixed#155]
### What Re:Fined version are you using? (versions other than listed are not supported)

Latest release

### Was the issue non-existant in some previous version?

2.90

### In which game is the bug happening?  (games other than listed are not supported)

Kingdom Hearts II Final Mix

### The bug & it's reproduction steps

Anytime you go into the "save/load" menu, it fails to check the system storage. When you're in-game and go to a save point, it fails once and returns you in-game. If you try to load a save from the title screen, it softlocks you and constantly pops up the error, completely unable to escape the loop without closing the game manually.
![image](https://github.com/KH-ReFined/KH-ReFined/assets/29827600/8385e3db-5cca-470e-b1fe-2122e16ca9fc)

The game also started getting frequent stutters every now and again, making the game feel like it's maxing out system resources (when it's not, it's just stuttering for no reason.) 

### What should've hapepned instead

I've used this mod since 1.0 and the save system worked just fine at the time. I tested it out by re-downloading 2.90 (mostly because 3.0 doesn't seem to work for me lol), and the save system works just fine. I've also tested the performance of 2.90 as well as re-tested the game without mods and it also saves and performs just fine. 

### Logs

[ReFined-10-02-2024.txt](https://github.com/KH-ReFined/KH-ReFined/files/14230242/ReFined-10-02-2024.txt)

### I did the following:

- [X] Make sure the Re:Fined .exe and OpenKH are up-to-date.
- [X] [See if the issue has been mentioned before](https://github.com/KH-ReFined/KH-ReFined/issues).

**Comment by TopazTK** [comment]
Both of these bugs do not exist anymore. Please make sure you have updated, and also delete the save as it may be corrupted.
Closing.

**Comment by Fivavoa** [comment]
Homie, these happen on the latest version. (I even included the titlebar where it says 4.50 for you and everything) so both of these persist. It even occurs when there are no saves. I don't know how you want me to update more than I already have.

**Comment by Fivavoa** [comment]
Everything seemed to work now, appreciate the assistance! Just figured that I would tell you one more thing, that at least for me, when I tried to have re:fined generate it's own folder that it's missing (as I just deleted it hoping that it would regenerate it's own "fixed" version on it's own", the game straight up isn't able to run again. It gets stuck at the boot sequence.
[ReFined-11-02-2024.txt](https://github.com/KH-ReFined/KH-ReFined/files/14231353/ReFined-11-02-2024.txt)

Simply having the folders already there was enough to get it to work, however.
